data/jobs.txt

```diff
@@ -2217,6 +2217,61 @@ mission "Prisoners [4]"
 			distance 3 4
 		fleet "pirate raid"
 
+mission "Released Prisoners [0]"
+	name "Released prisoner transport to <planet>"
```

Comment:
@Amazinite what do you think about "Released prisoner" -> "Parolee" for this series of jobs to shorten the names?